docs/setup/independent/install-kubeadm.md

```diff
@@ -14,11 +14,44 @@ This page shows how to use install kubeadm.
 * 1GB or more of RAM per machine (any less will leave little room for your apps)
 * Full network connectivity between all machines in the cluster (public or private network is fine)
 * Unique MAC address and product_uuid for every node
+* Certain ports are open on your machines. See the section below for more details
 
 {% endcapture %}
 
 {% capture steps %}
 
+## Check required ports
+
+### Master node(s)
+
+| Port Range | Purpose                         |
+|------------|---------------------------------|
+| 6443*      | Kubernetes API server           |
+| 2379-2380  | etcd server client API          |
+| 10250      | Kubelet API                     |
+| 10251      | kube-scheduler                  |
+| 10252      | kube-controller-manager         |
+| 10255      | Read-only Kubelet API (Heapster)|
+
+### Worker node(s)
+
+| Port Range  | Purpose                         |
+|-------------|---------------------------------|
+| 10250       | Kubelet API                     |
+| 10255       | Read-only Kubelet API (Heapster)|
+| 30000-32767 | Default port range for [NodePort Services](https://kubernetes.io/docs/concepts/services-networking/service). Typically, these ports would need to be exposed to external load-balancers, or other external consumers of the application itself. |
```

Comment:
Can you make the link relative? i.e.:

\[NodePort Services](/docs/concepts/services-networking/service/#type-nodeport)